packages/date-picker/src/DatePicker/components/Calendar/Calendar.scss

```diff
@@ -251,3 +251,18 @@ $rdp-cell-size: 40px;
   background-color: $color-blue-100;
   color: $color-blue-500;
 }
+
+// Add Storybook psuedo classes
+:global(.story__day-hover) {
+  &:not([aria-disabled="true"]) {
+    &:not(.outside) {
```

Comment:
We should probably revisit this `.outside` class and find out exactly what it's doing.